index.js

```diff
@@ -53,7 +53,15 @@ convert = function(source) {
           parentObj = parents.shift();
           break;
         default:
-          currentObj[currentKey] = currentValue;
+          if(currentObj[currentKey]) {
+            if(currentObj[currentKey].constructor === Array) {
+              currentObj[currentKey].push(currentValue);
+            } else {
+              currentObj[currentKey] = [currentObj[currentKey], currentValue];
+            }
```

Comment:
You can use `Array.isArray` to check if the variable is array.

Also, let's convert `currentObj[currentKey]` to `[currentObj[currentKey]]` and then just push `currentValue`

Something like this,
```
if (!Array.isArray(currentObj[currentKey])) {
  currentObj[currentKey] = [currentObj[currentKey]]
}
currentObj[currentKey].push(currentValue)
```

What do you think?